spaces/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+
+from .models import Address, Place
+
+
+class AddressSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Address
+        fields = ('street_name', 'civic', 'floor', 'door', 'city', 'postal_code', 'region', 'nation')
+
+
+class PlaceSerializer(serializers.HyperlinkedModelSerializer):
+    address = AddressSerializer()
+
+    class Meta:
+        model = Place
+        fields = ('url', 'name', 'owner', 'description', 'address')
+
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get('name', instance.name)
+        instance.owner = validated_data.get('owner', instance.owner)
+        instance.description = validated_data.get('nationality', instance.description)
+        instance.address = validated_data.get('address', instance.address)
+        instance.save()
+        return instance
+
+    def create(self, validated_data):
+        address_data = validated_data.get('address')
+        address = Address.objects.create(street_name=address_data.get('street_name'),
```

Comment:
You should never do this. You are replicating code from `address`, and it should not be the PlaceSerializer's responsability.
Just do `address = self.address.create(self, address_data)`

However, as I say in the previous note, this should already be handled by the HyperlinkedModelSerializer and this code might as well not exist.